repos/system_upgrade/el7toel8/actors/checktargetrepos/tests/test_checktargetrepos.py

```diff
@@ -76,7 +76,7 @@ def test_checktargetrepos_no_rhsm(monkeypatch, enable_repos, custom_target_repos
 
     if not custom_target_repos:
         assert reporting.create_report.called == 1
-        assert 'inhibitor' in reporting.create_report.report_fields.get('flags', [])
+        # assert 'inhibitor' in reporting.create_report.report_fields.get('flags', [])
```

Comment:
@Rezney why it's commented out? The inhibitor should be raised. if that's not true, update please the case studies table in the description of the actor and tests properly, to match the table.

Reply:
@pirat89 thanks for catching this left-over :)